Expensify/App
# Compare against null because sequenceNumber can be 0

### Details


### Fixed Issues

$ https://github.com/Expensify/App/issues/6233

### Tests / QA steps
1. Open app and login
2. Create a new chat by going + > New Chat and start chat with a user never had a conversation before
3. Type and send a single message
4. Long tap on it, mark as unread
5. Go back to LNH. Confirm its message preview is bolded to indicate its unread (unlike the issue video).

### Tested On


### Screenshots


#### Web


#### Mobile Web


#### Desktop


#### iOS


#### Android
